cluster-up/cluster/k8s-1.14.2-kind-sriov/manifests/sriovdp-daemonset.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: sriov-device-plugin
+  namespace: kube-system
+
+---
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: kube-sriov-device-plugin-amd64
+  namespace: kube-system
+  labels:
+    tier: node
+    app: sriovdp
+spec:
+  template:
+    metadata:
+      labels:
+        tier: node
+        app: sriovdp
+    spec:
+      nodeSelector:
+        beta.kubernetes.io/arch: amd64
```

Comment:
Don't we have to specify here the sriov label the select the proper node ?

Reply:
Not really. At the end this is going to run on a node that runs in a kind cluster that runs in a pod that was assigned via that selector. What we need to be sure of is that the pod that runs the test is assigned to a node equipped with sriov, but that's part of the Prow job configuration.